packages/core/src/features/st-var.ts

```diff
@@ -47,8 +54,8 @@ export const diagnostics = {
             .map((s, i) => (i === cyclicChain.length - 1 ? '↻ ' : i === 0 ? '→ ' : '↪ ') + s)
             .join('\n')}"`,
     MISSING_VAR_IN_VALUE: () => `invalid value() with no var identifier`,
-    COULD_NOT_RESOLVE_VALUE: (args: string) =>
-        `cannot resolve value function using the arguments provided: "${args}"`,
+    COULD_NOT_RESOLVE_VALUE: (args?: string) =>
```

Comment:
missing test case that expects any args

Reply:
This isn't a new diagnostics, it is tested there.